packages/web/src/scss/components/Tooltip/README.md

```diff
@@ -0,0 +1,190 @@
+# Tooltip
+
+Bare Tooltip HTML:
+
+```html
+<div class="Tooltip Tooltip--top">
+  Hello there!
+  <span class="Tooltip__arrow"></span>
+</div>
+```
+
+## Linking with Content
+
+To display a Tooltip along your content, simply place it in the DOM next to it.
+Tooltip is positioned relative to the closest parent element with
+`position: relative` or `position: absolute`. You may either provide the CSS
+yourself or you can use the prepared TooltipWrapper component.
+
+👉 **Basic Tooltip is static and doesn't react on user interaction.** Read
+further to see how to provide the desired behavior.
+
+👉 You don't need TooltipWrapper when your Tooltip is already being positioned
+correctly.
+
+👉 Depending on your layout, you may need to provide additional styling to
+shrink TooltipWrapper box model, e.g. using `d-inline-block` utility class.
+(This probably won't be necessary inside flexbox and grid layouts.)
+
+👉 Link Tooltip to your content using `id` and `aria-describedby` attributes for
+improved accessibility.
+
+```html
+<div class="TooltipWrapper d-inline-block">
+  <a href="#" aria-describedby="my-tooltip">I have a tooltip!</a>
+  <div id="my-tooltip" class="Tooltip Tooltip--top">
+    Hello there!
+    <span class="Tooltip__arrow"></span>
+  </div>
+</div>
+```
+
+## Placement
+
+Available placements are: top, right, bottom, and left. To apply the desired
+placement, just pick a corresponding CSS modifier: `Tooltip--top`,
+`Tooltip--right`, `Tooltip--bottom`, or `Tooltip--left`. For JS-controlled
+positioning please use the `data-placement` attribute instead (more on that
+below).
+
+```html
+<div class="Tooltip Tooltip--right">
+  Tooltip on right
+  <span class="Tooltip__arrow"></span>
+</div>
+```
+
+## Interaction
+
+### Pure CSS
+
+To display Tooltip on hover and focus (for focusable elements, i.e. mainly
+links, buttons or inputs), add `TooltipTarget` class to your element and place
+Tooltip HTML right after it.
+
+```html
+<div class="TooltipWrapper d-inline-block">
+  <a href="#" class="TooltipTarget" aria-describedby="my-tooltip-on-hover">I have a tooltip!</a>
+  <div id="my-tooltip-on-hover" class="Tooltip Tooltip--top">
+    Hello there!
+    <span class="Tooltip__arrow"></span>
+  </div>
+</div>
+```
```

Comment:
The idea here is to provide two ways, CSS and JS, because class name sometimes may not be accessible. But we can start with the CSS way and see if it's sufficient.

What does @crishpeen think?